components/TextFields/src/ContainedInputView/private/MDCTextControlStylePathDrawingUtils.h

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+@interface MDCTextControlStylePathDrawingUtils : NSObject
```

Comment:
Are we building something already offered by
https://github.com/material-components/material-components-ios/tree/develop/components/Shapes ?

which features were missing?

Reply:
I don't know, I've never looked at the Shapes library. If Shapes has similar helpers for adding specific corners to paths with corner radii I'm down to use those. Do such helpers exist?

Reply:
hmm im not sure. @yarneo may know more. 
For now Im ok with us adding a todo to look into shape integration.